templates/Includes/ResponsiveImageSet.ss

```diff
@@ -1,6 +1,8 @@
 <picture>
-    <% loop $Sizes %>
-        <source media="$Query" srcset="$Image.URL">
-    <% end_loop %>
-    <img src="$DefaultImage.URL" alt="$Title">
+	<!--[if IE 9]><video style="display: none;"><![endif]-->
```

Comment:
Maybe add a template comment or link to explain the IE9 conditional comment, or at least that this is the HTML suggested by the picturefill library. Minor, but it's a little confusing seeing this without looking at the picturefill docs.